### PR TITLE
feat: add scraper for SBCHC

### PR DESCRIPTION
**Pull request by tkjones117**
Based on similar MyChart scrapers (e.g. LynnTech), added a scraper for South Boston Community Health Center (SBCHC).

### CODE REVIEW

File: site-scrapers/SouthBostonCHC.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        PostDataCallback
+    );
+}
+
+/**
+ * This is the callback function. 
+ * The string was retrieved from the Form Data found in a GetOpeningsForProvider request.
+ */
+function PostDataCallback(startDateFormatted) {
+    return `id=${siteId}&vt=${1089}&dept=${dept}&view=grouped&start=${startDateFormatted}&filters=%7B%22Providers%22%3A%7B%221900119%22%3Atrue%7D%2C%22Departments%22%3A%7B%22150001007%22%3Atrue%7D%2C%22DaysOfWeek%22%3A%7B%220%22%3Atrue%2C%221%22%3Atrue%2C%222%22%3Atrue%2C%223%22%3Atrue%2C%224%22%3Atrue%2C%225%22%3Atrue%2C%226%22%3Atrue%7D%2C%22TimesOfDay%22%3A%22both%22%7D`;
```
Comment (adpjay) [inline site-scrapers/SouthBostonCHC.js:42]: Should the  `${1089}` be `${vt}`? Otherwise, it looks good!

Comment (tkjones117) [reply site-scrapers/SouthBostonCHC.js]: yes it should be - fixed!

**Comment by adpjay** [comment]
I just manually tested this and validated the scraped results against the South Boston CHC website. It looks good!

**Review by livgust — APPROVED** [review]
Approving based on approval from @adpjay

**Pull request merged**